ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_info.py

```diff
@@ -75,15 +75,15 @@
         "volumes",
         "volume_collections"
       - NOTE:-
```

Comment:
This section renders very ugly: https://datamattsson.github.io/nimble-ansible-modules/modules/hpe_nimble_info_module.html#hpe-nimble-info-module

Reply:
agreed but have no choice. :(

Reply:
Since this text blob doesn't allow advanced formatting or lists, why try cram it in? Can't we just write the description in plain English?

Reply:
so r u suggesting to remove NOTE:- completely??

can u please paste the text how you want it to be ?

Reply:
```
When supplied, this argument will define the information to be collected. Possible values for this include "all" "minimum" "config" "access_control_records", "alarms", "application_servers", "application_categories", "arrays", "chap_users", "controllers", "disks", "fibre_channel_interfaces", "fibre_channel_configs", "fibre_channel_initiator_aliases", "fibre_channel_ports", "folders", "groups", "initiator_groups", "initiators", "master_key", "network_configs", "performance_policies", "pools", "protection_schedules", "protection_templates", "protocol_endpoints", "replication_partners", "shelves", "snapshots", "snapshot_collections", "software_versions", "user_groups", "user_policies", "users", "volumes", "volume_collections".


Each subset except "all", "minimum" and "config" supports four types of subset options. Subset "all" supports limit and detail as subset options. Subset "config" and "minimum" does not support any subset options.


See the example section for usage of the following subset options.
"fields" - A string representing which attributes to display for a given subset.
"limit" - An integer value which represents how many latest items to show for a given subset.
"detail" - A bool flag when set to true fetches everything for a given subset. Default is "True".
"query" - A key-value pair to query.
```

Reply:
this will look even bad.. 


```
See the example section for usage of the following subset options.
"fields" - A string representing which attributes to display for a given subset.
"limit" - An integer value which represents how many latest items to show for a given subset.
"detail" - A bool flag when set to true fetches everything for a given subset. Default is "True".
"query" - A key-value pair to query.
```

will be shown in one line and not in a separate line.

- "fields" - A string representing which attributes to display for a given subset.  ===> will give rendering error. 
we cannot use double quote after '-' immediately.

let's leave it as it is..